demisto_sdk/tests/integration_tests/update_release_notes_integration_test.py

```diff
@@ -464,28 +464,33 @@ def test_update_release_notes_modified_apimodule(demisto_client, repo, mocker):
         taxii_feed_pack.path,
         "Integrations/FeedTAXII_integration/FeedTAXII_integration.yml",
     )
-    repo.id_set.update(
-        {
-            "scripts": [
-                {
-                    "ApiModules_script": {
-                        "name": "ApiModules_script",
-                        "file_path": api_module_script_path,
-                        "pack": "ApiModules",
-                    }
-                }
-            ],
-            "integrations": [
-                {
-                    "FeedTAXII_integration": {
-                        "name": "FeedTAXII_integration",
-                        "file_path": taxii_feed_integration_path,
-                        "pack": "FeedTAXII",
-                        "api_modules": ["ApiModules_script"],
-                    }
-                }
-            ],
-        }
+
+    # Mock the behavior of Neo4jContentGraphInterface
+    class MockedContentGraphInterface:
```

Comment:
**If** this is usable beyond just this task, consider moving to a shared file and importing from there, to avoid code duplication.